testing/iframe.js

```diff
@@ -299,6 +298,44 @@ export function createIframePromise(opt_runtimeOff, opt_beforeLayoutCallback) {
   });
 }
 
+/**
+ * @param {!Document} doc
+ * @param {string} cssText
+ * @param {function()} cb
+ */
+function installStylesLegacy(doc, cssText, cb) {
```

Comment:
I think the only thing calling this is `createIframePromise` which I guess is what we used before `describes.js` was a thing? Should we have an issue tracking removal of that function?

Reply:
I think the infra team is tracking the old `createIframePromise` removal. But it's a lower priority.